client/packages/common/src/ui/layout/tables/components/Cells/MultiNumberInputCell/MultiNumberInputCell.tsx

```diff
@@ -0,0 +1,167 @@
+/**
+ * Table cell component to present a numerical value as a combination of 2 (or
+ * more) units, e.g. Years/Months.
+ *
+ * Units are specified via the `units` prop. The smallest unit (e.g. "Months")
+ * should be the unit of the source "value", and the other units should be
+ * defined in relation to this, by specifying the ratio (the base unit has a
+ * ratio of 1). For example, for years and months we would define units like so:
+ *
+ * units={[
+ *         {
+ *           key: 'year',
+ *           ratio: 12,
+ *           label: "Y"
+ *         },
+ *         {
+ *           key: 'month',
+ *           ratio: 1,
+ *           label: "M"
+ *         },
+ *    ]}
+ *
+ * For hours/minutes/seconds:
+ *
+ * units={[
+ *         {
+ *           key: 'hour',
+ *           ratio: 3600,
+ *           label: "H"
+ *         },
+ *         {
+ *           key: 'minute',
+ *           ratio: 60,
+ *           label: "M"
+ *         },
+ *         {
+ *           key: 'second',
+ *           ratio: 1,
+ *           label: "S"
+ *         },
+ *    ]}
+ */
+
+import React from 'react';
+import { CellProps } from '../../../columns';
+import {
+  NumericInputProps,
+  NumericTextInput,
+  StandardTextFieldProps,
+} from '@common/components';
+import { Box } from '@openmsupply-client/common';
+import { RecordWithId } from '@common/types';
+import { useBufferState, useDebounceCallback } from '@common/hooks';
+
+interface Unit {
+  key: string;
+  ratio: number;
+  label: string;
+}
+
+export const MultipleNumberInputCell = <T extends RecordWithId>({
```

Comment:
Most of this is just copied straight from "NumberInputCell". Only changes are in relation to multi-unit functionality.